src/App.js

```diff
@@ -1,9 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import Footer from './components/Footer';
+import Header from './components/Header';
+import HeroEdit from './pages/HeroEdit'
+import HeroIndex from './pages/HeroIndex';
+import HeroNew from './pages/HeroNew';
+import HeroShow from './pages/HeroShow';
+import Home from './pages/Home';
+import NotFound from './pages/NotFound';
+import mockHeros from './mockHeros'
+import { Routes, Route } from "react-router-dom"
 
 const App = () => {
+
+  const [mockHero, setMockHero] = useState(mockHeros)
+
+  console.log(mockHero);
+
   return (
     <>
+      <Header />
+      <body>
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/heroindex" element={<HeroIndex />} />
+          <Route path="/heroshow" element={<HeroShow />} />
+          <Route path="/heronew" element={<HeroNew />} />
+          <Route path="/heroedit" element={<HeroEdit />} />
+          <Route path="*" element={<NotFound />} />
+        </Routes>
+        </body>
+      <Footer id="footer" />
```

Comment:
Hey  - remember this is React, so you don't need the body tags.  Also, you want any ids/classNames to be in the Footer.js component file, not directly in the component call.  Reach out if you have questions. Otherwise it looks great!